c7n/mu.py

```diff
@@ -348,7 +348,6 @@ def _create_or_update(self, func, role=None, s3_uri=None, qualifier=None):
 
             new_config = func.get_config()
             new_config['Role'] = role
-            del new_config['Runtime']
```

Comment:
Entered in c18b66351acc384d053ab782fcd7245705d8abd3, not clear why. Removing so config takes precedence over remote.